auth-api/pom.xml

```diff
@@ -188,6 +188,26 @@
       <artifactId>junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.dropwizard</groupId>
+      <artifactId>dropwizard-testing</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-core</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-junit-jupiter</artifactId>
```

Comment:
Is this artifact actually used?

Reply:
No it's not you are right. I tried to use the Mockito annotations but couldn't get it to work nicely with the dropwizward testing library so I scrapped it. Gonna remove it 👍